classes/core/PKPApplication.inc.php

```diff
@@ -194,83 +194,15 @@ public function initializeDatabaseConnection() {
-		Registry::set('laravelContainer', $laravelContainer);
-		(new Illuminate\Bus\BusServiceProvider($laravelContainer))->register();
-		(new Illuminate\Events\EventServiceProvider($laravelContainer))->register();
-		$eventDispatcher = new \Illuminate\Events\Dispatcher($laravelContainer);
-		$laravelContainer->instance('Illuminate\Contracts\Events\Dispatcher', $eventDispatcher);
-		$laravelContainer->instance('Illuminate\Contracts\Container\Container', $laravelContainer);
-
-		// Map valid config options to Illuminate database drivers
-		$driver = strtolower(Config::getVar('database', 'driver'));
-		if (substr($driver, 0, 8) === 'postgres') {
-			$driver = 'pgsql';
-		} else {
-			$driver = 'mysql';
-		}
-
-		$capsule = new Capsule;
-		$capsule->addConnection([
-			'driver'    => $driver,
-			'host'      => Config::getVar('database', 'host'),
-			'database'  => Config::getVar('database', 'name'),
-			'username'  => Config::getVar('database', 'username'),
-			'port'      => Config::getVar('database', 'port'),
-			'unix_socket'=> Config::getVar('database', 'unix_socket'),
-			'password'  => Config::getVar('database', 'password'),
-			'charset'   => Config::getVar('i18n', 'connection_charset', 'utf8'),
-			'collation' => Config::getVar('database', 'collation', 'utf8_general_ci'),
-		]);
-		$capsule->setEventDispatcher($eventDispatcher);
-		$capsule->setAsGlobal();
-		if (Config::getVar('database', 'debug')) Capsule::listen(function($query) {
-			error_log("Database query\n$query->sql\n" . json_encode($query->bindings));//\n Bindings: " . print_r($query->bindings, true));
-		});
-
-
-		// Set up Laravel queue handling
-		$laravelContainer->bind('exception.handler', function () {
-			return new class implements Illuminate\Contracts\Debug\ExceptionHandler {
-				public function shouldReport(Throwable $e) {
-					return true;
-				}
 
-				public function report(Throwable $e) {
-					error_log((string) $e);
-				}
-
-				public function render($request, Throwable $e) {
-					return null;
-				}
+		// Initialize Laravel's container and set it globally
+		import('lib.pkp.classes.core.PKPContainer');
+		$laravelContainer = new PKPContainer();
+		$laravelContainer->registerConfiguredProviders();
 
-				public function renderForConsole($output, Throwable $e) {
-					echo (string) $e;
-				}
-			};
-		});
-
-		$queue = new Illuminate\Queue\Capsule\Manager($laravelContainer);
-
-		// Synchronous (immediate) queue
-		$queue->addConnection(['driver' => 'sync']);
-
-		// Persistent queue
-		$connection = Capsule::schema()->getConnection();
-		$resolver = new \Illuminate\Database\ConnectionResolver(['default' => $connection]);
-		$manager = $queue->getQueueManager();
-		$laravelContainer['queue'] = $queue->getQueueManager();
-		$manager->addConnector('database', function () use ($resolver) {
-			return new Illuminate\Queue\Connectors\DatabaseConnector($resolver);
+		if (Config::getVar('database', 'debug')) \Illuminate\Support\Facades\DB ::listen(function($query) {
```

Comment:
Is the space between `DB` and `::` intentional?

Reply:
Oops